### PR TITLE
Add polling station intent

### LINKED ISSUE #17: Respond to questions about polling locations
Use should be able to ask questions about the polling location for a particular address.

### PR DESCRIPTION
**Pull request by jaumb**
closes #17

### CODE REVIEW

**Comment by jaumb** [comment]
@jmartini @hamstap85 ready for review when you have a chance

File: brooklinevoiceapp/mycity/intents/polling_stations_intent.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    LOGGER.debug('Getting polling location information')
+
+    response = MyCityResponseDataModel()
+    set_address_in_session(mycity_request)
+    current_address = \
+            mycity_request.session_attributes.get(intent_constants.CURRENT_ADDRESS_KEY)
+    if current_address is None:
+        # Delegate to the Alexa interaction model for getting the user address
+        LOGGER.debug('Requesting user address.')
+        response.dialog_directive = "Delegate"
```
Comment (jmartini) [inline brooklinevoiceapp/mycity/intents/polling_stations_intent.py]: We could alternatively use [auto delegation](https://developer.amazon.com/docs/custom-skills/delegate-dialog-to-alexa.html#automatically-delegate-simple-dialogs-to-alexa)

File: brooklinevoiceapp/mycity/intents/polling_stations_intent.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+        response.should_end_session = True
+
+    return response
+
+
+def _get_output_speech_for_address(address, mycity_request):
+    """
+    Creates output speech for polling locations near the provided address
+
+    :param address: Current address
```
Comment (jmartini) [inline brooklinevoiceapp/mycity/intents/polling_stations_intent.py]: Missing `:param mycity_request`

**Comment by Joshhw** [comment]
Closing since we already merged. Congrats @jaumb

**Pull request closed without merging**